docs/examples/Parameters/Parameters.ipynb

```diff
@@ -89,132 +107,206 @@
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 7,
-   "metadata": {
-    "ExecuteTime": {
-     "end_time": "2021-06-17T08:58:47.474820Z",
-     "start_time": "2021-06-17T08:58:47.464818Z"
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "dac:\n",
+      "\tparameter value\n",
+      "--------------------------------------------------------------------------------\n",
+      "IDN :\tNone \n",
+      "ch1 :\t0 (V)\n",
+      "ch2 :\t1 (V)\n"
+     ]
     }
-   },
-   "outputs": [],
+   ],
    "source": [
-    "class VirtualParameter(Parameter):\n",
-    "    def __init__(self, name, dac_param):\n",
-    "        self._dac_param = dac_param\n",
-    "        super().__init__(name)\n",
-    "        \n",
-    "    def get_raw(self):\n",
-    "        return self._dac_param.get()    "
+    "dac = DummyInstrument('dac', gates=['ch1', 'ch2'])\n",
+    "dac.ch2.set(1)\n",
+    "dac.print_readable_snapshot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then we create a DelegateParameter. Note that the DelegateParameter supports changing name, label, unit, scale and offset. Its therefor possible to use a DelegateParameter to perform a simple unit conversion."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 8,
-   "metadata": {
-    "ExecuteTime": {
-     "end_time": "2021-06-17T08:58:48.861043Z",
-     "start_time": "2021-06-17T08:58:48.853047Z"
-    }
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "vp1 = VirtualParameter('param_without_underlying_instrument_knowledge', dac.ch1)"
+    "my_delegate_param = DelegateParameter('my_delegated_parameter', dac.ch2, scale=1/1000, unit='mV')"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 9,
-   "metadata": {
-    "ExecuteTime": {
-     "end_time": "2021-06-17T08:58:49.967319Z",
-     "start_time": "2021-06-17T08:58:49.954319Z"
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "1000.0\n",
+      "mV\n"
+     ]
     }
-   },
-   "outputs": [],
+   ],
    "source": [
-    "vp1.underlying_instrument"
+    "print(my_delegate_param.get())\n",
+    "print(my_delegate_param.unit)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Right now, parameter `vp1` has no knowledge of the instrument of the parameter it wraps around. This can be addressed as follows."
+    "###  Manually creating a virtual parameter"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In some cases it may make sense to manually create a virtual parameter. In this example we will create a virtual parameter that abstracts channel 1 of a digital-to-analog converter (`dac`).\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Defining a virtual parameter\n",
```

Comment:
should this be level deeper? or perjaps these sub-heading are not needed at all since the "Manually creating a virtual parameter" is not long?
```suggestion
    "#### Defining a virtual parameter\n",
```